### PR TITLE
Improves logging

### PR DESCRIPTION
**Pull request by nwaldispuehl**
by:
- capitalizing log messages,
- adding additional logging, e.g. for user logins (session creation), and
- reducing the log level of these spammy grpc `info` lines to `debug`.

---

In our instance there are quite a lot of log lines like these:
```
INFO[0010]options.go:220 finished unary call with code OK              grpc.code=OK grpc.method=ListDevices grpc.service=proto.Devices grpc.start_time="2023-07-02T20:38:50+02:00" grpc.time_ms=0.258 span.kind=server system=grpc trace.id=a34950b0-b843-4844-b726-2b0abc543a70
```
By reducing their level to `debug` they are now filtered in the `info` setting.

What do you think?

### CODE REVIEW

**Review by fbuetler — CHANGES_REQUESTED** [review]
Nice :)

File: cmd/serve/main.go [hunk 1]
```diff
@@ -119,12 +119,12 @@ func (cmd *servecmd) Run() {
 		}
 		wgimpl, err := wgembed.NewWithOpts(wgOpts)
 		if err != nil {
-			logrus.Fatal(errors.Wrap(err, "failed to create wireguard interface"))
```
[the remark this answers is not in the document]
Comment (nwaldispuehl) [reply cmd/serve/main.go:122]: Awww, thanks a lot! :blush: 
As a usually german-speaking, usually Java-programming person, these lower case string are irritating at best... :smile_cat:

**Comment by nwaldispuehl** [comment]
Some unit tests are failing, but this appears to be an intermittent event outside of our control:
```
time="2023-07-03T10:49:02Z" level=warning msg="DNS lookup failed over TCP for upstream 185.150.99.255: read tcp 10.1.0.34:35558->185.150.99.255:53: i/o timeout"
```

**Comment by awlx** [comment]
This was actually a problem on my side I updated DNSDist on those hosts and this caused a restart loop ;). https://github.com/PowerDNS/pdns/issues/12975

**Review by awlx — APPROVED** [review]

**Review by fbuetler — APPROVED** [review]

**Pull request merged**